### PR TITLE
update proxy config yaml

### PR DESCRIPTION
**Pull request by xichengliudui**
Please provide a description for what this PR is for.
The new field is `ImageType`. `type` is no longer used

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [x] Configuration Infrastructure

### CODE REVIEW

**Comment by zirain** [comment]
this is not the right way to fix a reference page.
IIRC, you'd better send a PR in api repo, update the comments, then the automator will sync those changes here.

**Comment by zirain** [comment]
hold for https://github.com/istio/istio/issues/38968

**Review by ericvn — CHANGES_REQUESTED** [review]
This file is copied from the istio/api repo and needs to be changed there.

File: content/en/docs/reference/config/networking/proxy-config/index.html [hunk 1]
```diff
@@ -60,7 +60,7 @@
       app: ratings
   concurrency: 0
   image:
-    type: debug
+    imageType: debug
```
Comment (ericvn) [inline content/en/docs/reference/config/networking/proxy-config/index.html:63]: Example: This comes from  https://github.com/istio/api/blob/release-1.14/networking/v1beta1/proxy_config.pb.html#L60-L61

Comment (xichengliudui) [reply content/en/docs/reference/config/networking/proxy-config/index.html:63]: ok

Comment (xichengliudui) [reply content/en/docs/reference/config/networking/proxy-config/index.html:63]: waiting for confirmation to update API or update documentation

**Pull request closed without merging**